Bug Fix - remove trunk loading from patch-level `sum_fuel` variable

adrifoster opened the pull request:
As laid out in #1178, trunks should (but currently aren't) removed from the patch-level variable `sum_fuel` before being used to calculate rate of spread and fuel consumption.

See [Thonicke 2010](https://bg.copernicus.org/articles/7/1991/2010/bg-7-1991-2010.html) to confirm that this is how it is supposed to behave.

I added a quick fix at the end of the `charecteriscis_of_fuel` subroutine to do this.

### Description:

Just added one line to the end of the subroutine that removes the trunk litter. Currently this is not very nice looking, but since I am bringing in a big fire refactor I felt it was okay for now (??).

### Collaborators:
@rgknox @samsrabin 

### Expectation of Answer Changes:
Yes, if fire is turned on at all, this should change rate of spread calculations, and thus fire intensity, fuel consumption, tree mortality, etc....

### Checklist
*Contributor*
- [ x ] The in-code documentation has been updated with descriptive comments
- [ x ] The documentation has been assessed to determine if updates are necessary

*Integrator*
- [ ] FATES PASS/FAIL regression tests were run
- [ ] Evaluation of test results for answer changes was performed and results provided

### Documentation

I don't believe documentation updates are needed since we are updating the code to reflect what we actually intend.

### Test Results:

Working on this now...

*CTSM (or) E3SM (specify which) test hash-tag:*

*CTSM (or) E3SM (specify which) baseline hash-tag:*

*FATES baseline hash-tag:*

*Test Output:*

rgknox commented:
is sum_fuel used in other location(s) where it does want the trunks included? And if so, can you confirm that it is not used after the removal?

adrifoster commented:
> is sum_fuel used in other location(s) where it does want the trunks included? And if so, can you confirm that it is not used after the removal?

It is not used anywhere else where we would want it to be included. It is actually sort of corrected for already [here](https://github.com/NGEET/fates/blob/b054cd00ee3740f828b57b57768f8ec0bd1bc874/fire/SFMainMod.F90#L667):

```
      do c = 1,nfsc  
          tau_b(c)   =  39.4_r8 *(currentPatch%fuel_frac(c)*currentPatch%sum_fuel/0.45_r8/10._r8)* &
               (1.0_r8-((1.0_r8-currentPatch%burnt_frac_litter(c))**0.5_r8))  
       enddo
       tau_b(tr_sf)   =  0.0_r8
```

So I suppose `tau_b` and `tau_l` were already protected from this bug (directly), though not indirectly since the burned fraction depends on rate of spread, which is currently using the trunk fuels incorrectly.

samsrabin commented on fire/SFMainMod.F90:
```diff
@@ -306,6 +306,10 @@ subroutine  charecteristics_of_fuel ( currentSite )
+
+          ! remove trunks from patch%sum_fuel because they should not be included in fire equations
+          ! NOTE: ACF will update this soon to be more clean/bug-proof
+          currentPatch%sum_fuel = currentPatch%sum_fuel - litt_c%ag_cwd(4)
```
Would be safer to use tr_sf rather than 4, no?

adrifoster replied:
well....`tr_sf` is the trunks index in the SPITFIRE fuel array, not the litter array... though they are (implicitly) set up so that they are supposed to be the same order so I guess yes? I was going to fix this in the refactor

adrifoster commented:
Okay so this is really baffling me, but I am not seeing really any DIFFs from my tests:

`aux_clm`
no diffs

`fates`
`FAIL ERS_D_Ld30.f45_f45_mg37.I2000Clm50FatesCruRsGs.derecho_intel.clm-FatesColdLUH2 BASELINE fates-sci.1.72.2_api.34.0.0-ctsm5.1.dev175: DIFF`

I have re-done the tests multiple times and get the same answers so just to test that I am not crazy I added this (i.e. - 5000) to the relevant line that I added:

`currentPatch%sum_fuel = max(0.0, currentPatch%sum_fuel - litt_c%ag_cwd(tr_sf)) - 50000`

I re-tried one of the passing tests and it did in fact fail this time:

`FAIL SMS_D_Lm6_P256x1.f45_f45_mg37.I2000Clm50FatesRs.derecho_intel.clm-FatesCold BASELINE ctsm5.1.dev175: DIFF`

But the weird thing is that it only affects a handful of variables: FATES_FUEL_AMOUNT, FATES_ROS, FATES_FIRE_INTENSITY:

```
RMS FATES_FUEL_AMOUNT                4.7244E+04            NORMALIZED  2.0000E+00
 RMS FATES_ROS                        7.5491E+03            NORMALIZED  4.9849E+00
 RMS FATES_FIRE_INTENSITY             5.3122E+09            NORMALIZED  5.3534E+00
 RMS FATES_FUEL_AMOUNT                4.7244E+04            NORMALIZED  2.0000E+00
 RMS FATES_ROS                        7.6370E+03            NORMALIZED  4.9951E+00
 RMS FATES_FIRE_INTENSITY             5.2176E+09            NORMALIZED  5.3388E+00
 RMS FATES_FUEL_AMOUNT                4.7244E+04            NORMALIZED  2.0000E+00
 RMS FATES_ROS                        7.3927E+03            NORMALIZED  4.9725E+00
 RMS FATES_FIRE_INTENSITY             5.2050E+09            NORMALIZED  5.3908E+00
 RMS FATES_FUEL_AMOUNT                4.7244E+04            NORMALIZED  2.0000E+00
 RMS FATES_ROS                        7.1799E+03            NORMALIZED  4.9680E+00
 RMS FATES_FIRE_INTENSITY             5.2221E+09            NORMALIZED  5.3764E+00
 RMS FATES_FUEL_AMOUNT                4.7244E+04            NORMALIZED  2.0000E+00
 RMS FATES_ROS                        7.0678E+03            NORMALIZED  4.9582E+00
 RMS FATES_FIRE_INTENSITY             5.3114E+09            NORMALIZED  5.3721E+00
 RMS FATES_FUEL_AMOUNT                4.7245E+04            NORMALIZED  2.0000E+00
 RMS FATES_ROS                        7.2844E+03            NORMALIZED  5.0406E+00
 RMS FATES_FIRE_INTENSITY             5.4239E+09            NORMALIZED  5.3300E+00
 RMS FATES_FUEL_AMOUNT                4.7245E+04            NORMALIZED  2.0000E+00
 RMS FATES_ROS                        7.4843E+03            NORMALIZED  5.0384E+00
 RMS FATES_FIRE_INTENSITY             5.5129E+09            NORMALIZED  5.2384E+00
 RMS FATES_FUEL_AMOUNT                4.7245E+04            NORMALIZED  2.0000E+00
 RMS FATES_ROS                        7.7040E+03            NORMALIZED  4.9672E+00
 RMS FATES_FIRE_INTENSITY             5.5502E+09            NORMALIZED  5.2413E+00
 RMS FATES_FUEL_AMOUNT                4.7245E+04            NORMALIZED  2.0000E+00
 RMS FATES_ROS                        7.6608E+03            NORMALIZED  4.9305E+00
```

If these values are being updated so much I would think that other things would also be affected... I'm going to look at the output in xarray to see what is going on but if anyone else has any ideas please let me know...

adrifoster commented:
My one thought is that we don't have that many long tests so possibly we aren't running the model long enough to get a DIFF?

adrifoster commented:
But ultimately this is freaking me out that either there is another bug somewhere or that our tests do not actually cover enough

samsrabin commented:
Hmm. This change was only supposed to affect rate of spread, right? Not intensity / effects on vegetation? I guess if burned fraction is always either 0 or 1 both before and after the change, that would explain why only the output metrics are affected. 

I don't expect that to be the case for a 6-month run though. Unless maybe it's from a cold start and there's just not that much woody fuel?

It's also really weird that it only failed on a re-run!

adrifoster commented:
Incidentally I'm also now running a +50000 test...

ekluzek commented:
@samsrabin weird problems like this always deserve having someone else replicate the results. @adrifoster has already rerun several times, so if you (or someone else) could replicate the results that would be one good step to try.

Also @adrifoster what is unique about this test from others? And can you show it with a smaller grid? Is this really the longest test at 30 days for FATES?

adrifoster commented:
> Also @adrifoster what is unique about this test from others? And can you show it with a smaller grid? Is this really the longest test at 30 days for FATES?

Well this is a 6 month test... but yes I *think* (?) this is the longest FATES test we have.

adrifoster commented:
> Incidentally I'm also now running a +5000 test...

Okay...that one produced big diffs across all the  variables I expected:

```
 RMS FATES_LBLAYER_COND               6.7596E-02            NORMALIZED  6.9324E-02
 RMS FATES_LEAFC                      1.9652E-02            NORMALIZED  8.9338E-01
 RMS FATES_LEAF_ALLOC                 1.2992E-09            NORMALIZED  6.2594E-01
 RMS FATES_LITTER_IN                  9.5804E-08            NORMALIZED  2.0296E+01
 RMS FATES_LITTER_OUT                 3.8456E-10            NORMALIZED  5.9907E-01
 RMS FATES_MAINT_RESP                 3.2402E-09            NORMALIZED  9.8429E-01
 RMS FATES_MORTALITY_CFLUX_CANOPY     1.6034E-07            NORMALIZED  2.7380E+01
 RMS FATES_MORTALITY_CFLUX_USTORY     5.2101E-11            NORMALIZED  6.7528E+01
 RMS FATES_NCHILLDAYS                 5.0783E-01            NORMALIZED  7.5223E-03
 RMS FATES_NCOHORTS                   5.1768E+01            NORMALIZED  5.9666E-01
 RMS FATES_NEP                        9.6554E-09            NORMALIZED  8.0159E-01
 RMS FATES_NESTEROV_INDEX             9.5115E+02            NORMALIZED  1.2862E-01
 RMS FATES_NONSTRUCTC                 6.0521E-02            NORMALIZED  9.3498E-01
 RMS FATES_NPP                        9.5666E-09            NORMALIZED  6.9367E-01
 RMS FATES_PRIMARY_PATCHFUSION_ERR    3.1389E-14            NORMALIZED  4.7221E+00
 RMS FATES_ROS                        1.6502E+02            NORMALIZED  5.7802E+00
 RMS FATES_SAPWOODC                   2.1369E-02            NORMALIZED  1.0910E+00
 RMS FATES_SEEDS_IN                   2.0842E-09            NORMALIZED  9.2740E-01
 RMS FATES_SEED_ALLOC                 2.0318E-09            NORMALIZED  8.4177E-01
 RMS FATES_SEED_BANK                  5.3020E-03            NORMALIZED  3.7570E-01
 RMS FATES_STEM_ALLOC                 2.4038E-09            NORMALIZED  9.8285E-01
 RMS FATES_STOMATAL_COND              3.9656E-02            NORMALIZED  7.4756E-01
 RMS FATES_STOREC                     2.8022E-02            NORMALIZED  9.3331E-01
 RMS FATES_STORE_ALLOC                2.3735E-09            NORMALIZED  7.4594E-01
 RMS FATES_STRUCTC                    2.2832E-02            NORMALIZED  1.0843E+00
 RMS FATES_TRIMMING                   1.3010E-01            NORMALIZED  1.3736E-01
 RMS FATES_USTORY_VEGC                4.5092E-04            NORMALIZED  6.7528E+01
 RMS FATES_VEGC                       1.1090E-01            NORMALIZED  9.5760E-01
 RMS FATES_VEGC_ABOVEGROUND           6.2520E-02            NORMALIZED  9.7083E-01
 RMS FATES_ERROR_EL                   1.4624E-17            NORMALIZED  2.0837E+00
 RMS FATES_FIRE_FLUX_EL               6.5194E-08            NORMALIZED  3.3163E+01
 RMS FATES_PATCHAREA_LU               3.2318E-17            NORMALIZED  1.6159E-16
 RMS FATES_DISTURBANCE_RATE_MATRIX_LU 3.4704E+01            NORMALIZED  2.0790E+01
 RMS FATES_AREA_PLANTS                2.4197E-01            NORMALIZED  8.4616E-01
 RMS FATES_AREA_TREES                 2.5702E-01            NORMALIZED  1.0127E+00
 RMS FATES_AUTORESP                   4.2424E-09            NORMALIZED  8.2155E-01
 RMS FATES_AUTORESP_CANOPY            4.2377E-09            NORMALIZED  8.2072E-01
 RMS FATES_AUTORESP_USTORY            3.5150E-11            NORMALIZED  6.7528E+01
 RMS FATES_BURNFRAC                   5.5035E-08            NORMALIZED  4.1618E+00
 RMS FATES_CANOPY_VEGC                1.1171E-01            NORMALIZED  9.6040E-01
 RMS FATES_CBALANCE_ERROR             1.4345E-17            NORMALIZED  1.9698E+00
 RMS FATES_COLD_STATUS                5.1299E-02            NORMALIZED  2.7031E-02
 RMS FATES_CROOT_ALLOC                1.7126E-09            NORMALIZED  1.0372E+00
 RMS FATES_DAYSINCE_COLDLEAFOFF       1.0679E-01            NORMALIZED  7.1209E-04
 RMS FATES_DAYSINCE_COLDLEAFON        3.9956E+00            NORMALIZED  6.2707E-02
 RMS FATES_DEMOTION_CARBONFLUX        2.2000E-09            NORMALIZED  6.7528E+01
 RMS FATES_DISTURBANCE_RATE_FIRE      1.7356E+02            NORMALIZED  4.1618E+00
 RMS FATES_DISTURBANCE_RATE_TREEFALL  7.1921E-02            NORMALIZED  2.6020E+00
 RMS FATES_EFFECT_WSPEED              9.8143E-01            NORMALIZED  5.0253E-01
 RMS FATES_FDI                        7.4054E-03            NORMALIZED  2.3528E-02
 RMS FATES_FIRE_CLOSS                 5.0247E-08            NORMALIZED  3.7407E+01
 RMS FATES_FIRE_INTENSITY             4.9914E+07            NORMALIZED  5.7306E+00
 RMS FATES_FIRE_INTENSITY_BURNFRAC    5.9785E+05            NORMALIZED  6.2696E+00
 RMS FATES_FROOTC                     2.0012E-02            NORMALIZED  8.6201E-01
 RMS FATES_FROOT_ALLOC                1.5734E-09            NORMALIZED  6.7765E-01
 RMS FATES_FUELCONSUMED               9.6544E-03            NORMALIZED  4.1401E+00
 RMS FATES_FUEL_AMOUNT                4.1435E+04            NORMALIZED  2.2735E+00
 RMS FATES_FUEL_BULKD                 3.2980E+00            NORMALIZED  6.4103E-01
 RMS FATES_FUEL_EFF_MOIST             6.5743E-02            NORMALIZED  1.0355E-01
 RMS FATES_FUEL_MEF                   1.3012E-01            NORMALIZED  5.5406E-01
 RMS FATES_FUEL_SAV                   2.6554E+03            NORMALIZED  5.2085E-01
 RMS FATES_GDD                        1.0524E+01            NORMALIZED  8.4521E-01
 RMS FATES_GPP                        1.3027E-08            NORMALIZED  6.9708E-01
 RMS FATES_GPP_CANOPY                 1.3014E-08            NORMALIZED  6.9640E-01
 RMS FATES_GPP_USTORY                 9.6296E-11            NORMALIZED  6.7528E+01
 RMS FATES_GROWTH_RESP                1.2646E-09            NORMALIZED  6.8226E-01
 RMS FATES_HET_RESP                   2.4874E-10            NORMALIZED  5.5350E-02
 RMS FATES_IGNITIONS                  4.1581E-14            NORMALIZED  4.2210E+00
 RMS FATES_LBLAYER_COND               6.8911E-02            NORMALIZED  7.1293E-02
 RMS FATES_LEAFC                      1.9769E-02            NORMALIZED  8.9490E-01
 RMS FATES_LEAF_ALLOC                 1.3348E-09            NORMALIZED  6.6266E-01
 RMS FATES_LITTER_IN                  7.3855E-08            NORMALIZED  1.9830E+01
 RMS FATES_LITTER_OUT                 3.8441E-10            NORMALIZED  5.9416E-01
 RMS FATES_MAINT_RESP                 3.2626E-09            NORMALIZED  9.8560E-01
 RMS FATES_MORTALITY_CFLUX_CANOPY     1.2368E-07            NORMALIZED  2.9194E+01
 RMS FATES_MORTALITY_CFLUX_USTORY     5.3868E-11            NORMALIZED  6.7528E+01
 RMS FATES_NCHILLDAYS                 5.0956E-01            NORMALIZED  7.5293E-03
 RMS FATES_NCOHORTS                   5.2065E+01            NORMALIZED  5.9893E-01
 RMS FATES_NEP                        9.6899E-09            NORMALIZED  7.9770E-01
 RMS FATES_NESTEROV_INDEX             9.5014E+02            NORMALIZED  1.2723E-01
 RMS FATES_NONSTRUCTC                 6.0972E-02            NORMALIZED  9.3794E-01
 RMS FATES_NPP                        9.6090E-09            NORMALIZED  6.9249E-01
 RMS FATES_PRIMARY_PATCHFUSION_ERR    3.1412E-14            NORMALIZED  4.8421E+00
 RMS FATES_ROS                        1.6620E+02            NORMALIZED  5.6946E+00
 RMS FATES_SAPWOODC                   2.1593E-02            NORMALIZED  1.0961E+00
 RMS FATES_SEEDS_IN                   2.0511E-09            NORMALIZED  9.1200E-01
 RMS FATES_SEED_ALLOC                 2.0064E-09            NORMALIZED  8.3057E-01
 RMS FATES_SEED_BANK                  5.3643E-03            NORMALIZED  3.7558E-01
 RMS FATES_STEM_ALLOC                 2.5688E-09            NORMALIZED  1.0372E+00
 RMS FATES_STOMATAL_COND              4.0371E-02            NORMALIZED  7.5817E-01
 RMS FATES_STOREC                     2.8211E-02            NORMALIZED  9.3528E-01
 RMS FATES_STORE_ALLOC                2.3786E-09            NORMALIZED  7.6540E-01
 RMS FATES_STRUCTC                    2.3071E-02            NORMALIZED  1.0895E+00
 RMS FATES_TRIMMING                   1.3100E-01            NORMALIZED  1.3838E-01
 RMS FATES_USTORY_VEGC                6.4740E-04            NORMALIZED  6.7528E+01
 RMS FATES_VEGC                       1.1178E-01            NORMALIZED  9.6077E-01
 RMS FATES_VEGC_ABOVEGROUND           6.3026E-02            NORMALIZED  9.7390E-01
 RMS FATES_ERROR_EL                   1.4345E-17            NORMALIZED  1.9698E+00
 RMS FATES_FIRE_FLUX_EL               5.0247E-08            NORMALIZED  3.7407E+01
 RMS FATES_PATCHAREA_LU               3.1367E-17            NORMALIZED  1.5684E-16
 RMS FATES_DISTURBANCE_RATE_MATRIX_LU 3.4704E+01            NORMALIZED  2.0790E+01
```


Okay, so next thing... is this just a product of not running the model long enough (from bare ground - like all our tests) to get enough fuel to burn? Or is there an actual issue here. 

I suppose the thing to do would be to re-run an original test (without the crazy +/- 5000) for a longer time (20 years?) and see if this produces the expected DIFFs. I will do that (first I need to make a baseline for it).

@samsrabin if you have time to replicate some of this just to make sure I am not missing something I am happy to collaborate, but otherwise we can discuss tomorrow at the SE meeting...

adrifoster commented:
> > Also @adrifoster what is unique about this test from others? And can you show it with a smaller grid? Is this really the longest test at 30 days for FATES?
> 
> Well this is a 6 month test... but yes I _think_ (?) this is the longest FATES test we have.

Okay no I am wrong we do have a 25 month test... I guess even that isn't long enough. Will look at that to see.
`ERS_P128x1_Lm25.f10_f10_mg37.I2000Clm51Fates.derecho_intel.clm-FatesColdNoComp.C.0326-204943de_int`

samsrabin commented:
I think what you said about the baseline not having any `FIRE_CLOSS` makes everything start to click into place. The -50000 test didn't have any effect because there wasn't any burning to begin with, because there wasn't enough fuel. Then when you did the +50000 test, that makes it so there _is_ actually burning, thus affecting all those variables.

So yes, re-running for 20 years would be a good idea. You could even do it at 10x15 rather than 4x5 if you want. Let's discuss at the SE meeting tomorrow, and if there's more weirdness we can discuss how I might be able to help.

Some side notes:
- The `ERS_D_Ld30.f45_f45_mg37.I2000Clm50FatesCruRsGs.derecho_intel.clm-FatesColdLUH2` test _did_ have diffs without the ±50000 code. What's different about it that (presumably?) allowed fire to occur within 30 days of a cold start? Does it have something to do with LUH2?
- It sounds like we need longer FATES tests, at least in the `fates` suite. How long is necessary for fire to really get going?

adrifoster commented:
Yes totally agree on the LUH2 test being the outlier and the key... I will look into that test.

I also think we may not definitely need a longer test, but we could get around it with a test with initialized (cohorts and fuel I suppose) data.

@ekluzek and I have been discussing this but I think 2 tests that would be great to have are:

1) BCI single-point test with initialized data
2) NLDAS2 (conterminous US) test  with cohorts initialized from FIA data. I think we can also get (some) fuel information from FIA too.

samsrabin commented:
Oh yeah, using pre-initialized data is a great idea!

ekluzek commented:
For CTSM issues that address this are:

https://github.com/ESCOMP/CTSM/issues/2159
https://github.com/ESCOMP/CTSM/issues/1277

glemieux commented:
> For CTSM issues that address this are:
> 
> [ESCOMP/CTSM#2159](https://github.com/ESCOMP/CTSM/issues/2159) [ESCOMP/CTSM#1277](https://github.com/ESCOMP/CTSM/issues/1277)

Also note https://github.com/NGEET/fates/issues/846 for the fates-side issue

adrifoster commented:
Okay so a 10-year test showed DIFFs we expect:

`SMS_Lm125.f10_f10_mg37.I2000Clm50FatesRs.derecho_intel.clm-FatesCold`
```
 RMS FATES_STEM_ALLOC                 4.9766E-10            NORMALIZED  6.5747E-02
 RMS FATES_STOMATAL_COND              6.6146E-04            NORMALIZED  1.7434E-02
 RMS FATES_STOREC                     2.3615E-03            NORMALIZED  1.7685E-02
 RMS FATES_STORE_ALLOC                2.3800E-09            NORMALIZED  2.4524E-01
 RMS FATES_STRUCTC                    2.2289E-02            NORMALIZED  2.4879E-02
 RMS FATES_TRIMMING                   2.7334E-02            NORMALIZED  3.3526E-02
 RMS FATES_USTORY_VEGC                2.3357E-02            NORMALIZED  4.7752E-02
 RMS FATES_VEGC                       5.0689E-02            NORMALIZED  2.5741E-02
 RMS FATES_VEGC_ABOVEGROUND           2.9684E-02            NORMALIZED  2.5860E-02
 RMS FATES_ERROR_EL                   2.2160E-16            NORMALIZED  1.9724E+00
 RMS FATES_FIRE_FLUX_EL               2.4077E-11            NORMALIZED  1.1678E+00
 RMS FATES_PATCHAREA_LU               4.7836E-17            NORMALIZED  2.3918E-16
 RMS FATES_DISTURBANCE_RATE_MATRIX_LU 4.0403E-03            NORMALIZED  1.6679E-02
 RMS FATES_AREA_PLANTS                2.2989E-04            NORMALIZED  3.2167E-04
 RMS FATES_AREA_TREES                 6.9124E-03            NORMALIZED  1.5595E-02
 RMS FATES_AUTORESP                   2.5550E-10            NORMALIZED  1.5285E-02
 RMS FATES_AUTORESP_CANOPY            2.9581E-10            NORMALIZED  2.4240E-02
 RMS FATES_AUTORESP_USTORY            9.5013E-11            NORMALIZED  2.1055E-02
 RMS FATES_BURNFRAC                   1.3175E-09            NORMALIZED  1.5042E-02
 RMS FATES_CANOPY_SPREAD              4.8912E-03            NORMALIZED  6.8447E-03
 RMS FATES_CANOPY_VEGC                4.1104E-02            NORMALIZED  2.7758E-02
 RMS FATES_CBALANCE_ERROR             2.2847E-16            NORMALIZED  2.1780E+00
 RMS FATES_CROOT_ALLOC                2.5589E-10            NORMALIZED  5.1991E-02
 RMS FATES_DAYSINCE_COLDLEAFON        6.9171E-02            NORMALIZED  4.2650E-05
 RMS FATES_DEMOTION_CARBONFLUX        1.3551E-07            NORMALIZED  4.1363E+00
 RMS FATES_DISTURBANCE_RATE_FIRE      5.1961E-02            NORMALIZED  9.7602E-03
 RMS FATES_DISTURBANCE_RATE_TREEFALL  4.8319E-04            NORMALIZED  1.5513E-02
 RMS FATES_EFFECT_WSPEED              1.0673E-02            NORMALIZED  4.8259E-03
 RMS FATES_FDI                        1.6774E-04            NORMALIZED  6.3593E-04
 RMS FATES_FIRE_CLOSS                 3.5277E-11            NORMALIZED  8.9859E-01
 RMS FATES_FIRE_INTENSITY             3.6101E+04            NORMALIZED  4.2111E-02
 RMS FATES_FIRE_INTENSITY_BURNFRAC    9.7221E+03            NORMALIZED  9.6984E-02
 RMS FATES_FROOTC                     2.5293E-03            NORMALIZED  2.2640E-02
 RMS FATES_FROOT_ALLOC                9.3623E-10            NORMALIZED  1.9272E-01
 RMS FATES_FUELCONSUMED               3.5973E-03            NORMALIZED  3.0541E-02
 RMS FATES_FUEL_AMOUNT                2.5664E-01            NORMALIZED  2.4862E-01
 RMS FATES_FUEL_BULKD                 2.8459E-02            NORMALIZED  3.2638E-03
 RMS FATES_FUEL_EFF_MOIST             1.1886E-03            NORMALIZED  1.6486E-03
 RMS FATES_FUEL_MEF                   4.6055E-04            NORMALIZED  1.4652E-03
 RMS FATES_FUEL_SAV                   1.2173E+01            NORMALIZED  2.7888E-03
 RMS FATES_GPP                        7.1042E-10            NORMALIZED  1.2573E-02
 RMS FATES_GPP_CANOPY                 7.5769E-10            NORMALIZED  1.6517E-02
 RMS FATES_GPP_USTORY                 2.8702E-10            NORMALIZED  2.6999E-02
 RMS FATES_GROWTH_RESP                7.0728E-11            NORMALIZED  1.2719E-02
 RMS FATES_HET_RESP                   3.2853E-10            NORMALIZED  2.2204E-02
 RMS FATES_IGNITIONS                  2.7709E-16            NORMALIZED  1.9092E-02
 RMS FATES_LBLAYER_COND               2.6303E-03            NORMALIZED  2.4270E-03
 RMS FATES_LEAFC                      1.1445E-03            NORMALIZED  1.3650E-02
 RMS FATES_LEAF_ALLOC                 8.8520E-10            NORMALIZED  1.5540E-01
 RMS FATES_LITTER_IN                  9.7858E-10            NORMALIZED  7.6169E-02
 RMS FATES_LITTER_OUT                 4.1767E-10            NORMALIZED  2.0731E-02
 RMS FATES_MAINT_RESP                 2.0403E-10            NORMALIZED  1.8290E-02
 RMS FATES_MORTALITY_CFLUX_CANOPY     5.1375E-11            NORMALIZED  2.5755E-02
 RMS FATES_MORTALITY_CFLUX_USTORY     1.9477E-08            NORMALIZED  7.5365E-02
 RMS FATES_NCHILLDAYS                 1.3834E-01            NORMALIZED  1.5078E-03
 RMS FATES_NCOHORTS                   1.5864E+01            NORMALIZED  3.0569E-02
 RMS FATES_NEP                        6.7610E-10            NORMALIZED  2.0507E-02
 RMS FATES_NESTEROV_INDEX             1.8219E+00            NORMALIZED  1.1730E-04
 RMS FATES_NONSTRUCTC                 3.8203E-02            NORMALIZED  4.0620E-02
 RMS FATES_NPP                        5.1303E-10            NORMALIZED  1.2472E-02
 RMS FATES_PRIMARY_PATCHFUSION_ERR    1.0856E-14            NORMALIZED  1.3825E+00
 RMS FATES_PROMOTION_CARBONFLUX       1.2893E-07            NORMALIZED  9.3507E+00
 RMS FATES_ROS                        3.6405E-04            NORMALIZED  9.9060E-03
 RMS FATES_SAPWOODC                   3.5117E-02            NORMALIZED  4.7141E-02
 RMS FATES_SEEDS_IN                   3.1506E-09            NORMALIZED  3.2669E-01
 RMS FATES_SEED_ALLOC                 3.1505E-09            NORMALIZED  3.2631E-01
 RMS FATES_SEED_BANK                  1.5137E-02            NORMALIZED  4.9994E-02
 RMS FATES_STEM_ALLOC                 3.8384E-10            NORMALIZED  5.1991E-02
 RMS FATES_STOMATAL_COND              6.2061E-04            NORMALIZED  1.6474E-02
 RMS FATES_STOREC                     2.3207E-03            NORMALIZED  1.7299E-02
 RMS FATES_STORE_ALLOC                1.3289E-09            NORMALIZED  1.3784E-01
 RMS FATES_STRUCTC                    2.2264E-02            NORMALIZED  2.4842E-02
 RMS FATES_TRIMMING                   2.7324E-02            NORMALIZED  3.3518E-02
 RMS FATES_USTORY_VEGC                2.0540E-02            NORMALIZED  4.1927E-02
 RMS FATES_VEGC                       5.0711E-02            NORMALIZED  2.5730E-02
 RMS FATES_VEGC_ABOVEGROUND           2.9696E-02            NORMALIZED  2.5844E-02
 RMS FATES_ERROR_EL                   2.2847E-16            NORMALIZED  2.1780E+00
 RMS FATES_FIRE_FLUX_EL               3.5277E-11            NORMALIZED  8.9859E-01
 RMS FATES_PATCHAREA_LU               5.0825E-17            NORMALIZED  2.5412E-16
 RMS FATES_DISTURBANCE_RATE_MATRIX_LU 1.0396E-02            NORMALIZED  4.8532E-02
```

rgknox commented:
I don't think so, looks good, lets merge
